assets/platforms/fortinet.yaml

```diff
@@ -0,0 +1,40 @@
+---
+platform-type: 'fortinet'
+default:
+  driver-type: 'network'
+  privilege-levels:
+    exec:
+      name: 'exec'
+      pattern: '(?im)^[\w\._- ]+@[\w\.\(\)_-]+#\s?'
+      previous-priv:
+      deescalate:
+      escalate:
+      escalate-auth: false
+      escalate-prompt:
+    # configuration:
```

Comment:
unneeded? can we remove or uncomment if its supposed to be there?

Reply:
seems foritnet has previledge mode too, let me do more vestigation, pending this one platform